pyperformance/data-files/benchmarks/bm_asyncio_websockets/run_benchmark.py

```diff
@@ -23,13 +24,20 @@ async def handler(websocket) -> None:
     stop.set()
 
 
+async def send(ws):
+    try:
+        await ws.send(DATA)
+    except websockets.exceptions.ConnectionClosedOK:
```

Comment:
Does it happen on windows only or other platforms too?

Reply:
Windows only.